Common/ColorConv.cpp

```diff
@@ -408,7 +408,7 @@ void ConvertRGBA4444ToRGBA8888(u32 *dst32, const u16 *src, const u32 numPixels)
 		__m128i b = _mm_and_si128(_mm_srli_epi16(c, 8), mask4);
 		// And lastly 00A0 00A0.  No mask needed, we have a wall.
 		__m128i a = _mm_srli_epi16(c, 12);
-		a = _mm_slli_epi16(g, 8);
+		a = _mm_slli_epi16(a, 8);
```

Comment:
So, going in the order of the bits, we start out with ABGR, and we're aiming to result in AABBGGRR.  To do this, it creates 0A0B0G0R, and then or's to get AABBGGRR.

Specifically, here, we shift A right 12 against a wall - obtaining 000A.  We then need to shift it left 8 to get 0A00 (in bit order.)  So this change is correct as is, AFAICT.  Otherwise ` _mm_or_si128(b, a)` would overlap B and A on the same bits and not fill A.

-[Unknown]